cmd/deploy.go

```diff
@@ -112,24 +112,9 @@ func newDeployCmd() *cobra.Command {
 func (dc *deployCmd) validateArgs(cmd *cobra.Command, args []string) error {
 	var err error
 
-	dc.locale, err = i18n.LoadTranslations()
```

Comment:
Why are we getting rid of this? The `loadAPIModel` method is still using this private property:

```
apiloader := &api.Apiloader{
 		Translator: &i18n.Translator{
 			Locale: dc.locale,
 		},
 	}
```

Reply:
it's done in the `dc.validate()`, which is really `dc.generateCmd.validate()` just below.
it should probably be moved out of validate and into the creation of the cmds.